src/components/RequiresBLE/BluetoothDisabled.js

```diff
@@ -13,6 +13,12 @@ import { deviceNames } from "../../wording";
 
 function BluetoothDisabled() {
   const { colors } = useTheme();
+
+  useEffect(() => {
+    // Prompts the user to enable bluetooth using native api calls when the component gets initially rendered.
+    NativeModules.BluetoothHelperModule.prompt();
+  });
```

Comment:
does it work without the dependencies array? :thinking: 
I think you want to trigger this once on mount just add an empty dependency array.

Reply:
I forgot it (I'm used to having the linter complain) 🤦. It does work, but only by chance I guess since the component does not re-render.